infrastructure/httpapi/handlers/crossfire.go

```diff
@@ -136,6 +136,13 @@ func (handler *Crossfire) getValidatorStats(operatorAddress string, dbValidator
 			finalValidatorStats.CommitCountPhase2 = validatorStatRow.Value
 		case crossfire_constants.PHASE_1N2_COMMIT_PREFIX:
 			finalValidatorStats.CommitCountPhase1n2 = validatorStatRow.Value
+		case crossfire_constants.VOTED_PROPOSAL_ID:
+			{
+				finalValidatorStats.TaskVote = crossfire_constants.INCOMPLETED
```

Comment:
If the participant does not vote, there will be no row inside validator stat right? So this switch case will never reached, and the TaskVote will not be `INCOMPLETED`?

Reply:
@cdc-Hitesh

Reply:
You are absolutely correct. Spot on. I am changing this.

Also @calvinaco  I happen to find one another interesting issue. Pushing the changes. Will update you